t/05-scheduler-dependencies.t

```diff
@@ -1026,6 +1026,48 @@ while (my ($k, $v) = each %tests) {
     subtest "$k ordered" => $v;
 }
 
+subtest "SAP setup - issue 52928" => sub {
```

Comment:
Wouldn't it make sense to describe the particularity of this setup? At least I'm not familiar with what SAP makes so special.

Reply:
It's not SAP specific at all - it's just random dependencies. Take SAP as a nick name for a strange setup :)